torchmultimodal/diffusion_labs/models/adm_unet/adm.py

```diff
@@ -471,8 +507,8 @@ def adm_unet(
     Args:
         time_embed_dim (int): desired dimensionality of timestep embedding
         cond_embed_dim (int): desired dimensionality of conditional input embeddings
-        clip_embed_dim (int): expected dimensionality of CLIP image embeddings
-        clip_embed_name (str): name of CLIP embedding conditional input
+        embed_dim (int): expected dimensionality of CLIP image embeddings
```

Comment:
The description here shouldn't say CLIP either. It's the expected dim of the conditional image embedding